sdcm/nemesis.py

```diff
@@ -3864,6 +3864,10 @@ def decommission_nodes(self, add_nodes_number, rack, is_seed: Optional[Union[boo
                           f'{str(exc)}').publish()
 
     def disrupt_grow_shrink_cluster(self):
+        sleep_time_between_ops = self.cluster.params.get('nemesis_sequence_sleep_between_ops')
```

Comment:
@Mark-Gurevich , because of this change, please run using this branch the individual nemesis GrowShrinkCluster, to be sure it won't be affected by it

Reply:
ok